.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
   - DOCKER_TAG=ubuntu-18.04
   - DOCKER_TAG=fedora-30
   - DOCKER_TAG=opensuse-leap
+  - DOCKER_TAG=ubuntu-18.04 TPM2_TSS_VERSION=master
```

Comment:
this is a really good idea, we can set the minimum stable version and be building against master as we go. Im struggling if we want both gcc and clang builds or just add this as an include build and generate only one build, but I guess we do indeed want to know if something weird with clang or gcc pops up.